multimedia/v4l2tools/Makefile

```diff
@@ -0,0 +1,38 @@
+#
+# Copyright (C) 2017 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=v4l2tools
+PKG_VERSION:=0.1.5
+PKG_RELEASE:=$(AUTORELEASE)
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/mpromonet/v4l2tools.git
+PKG_SOURCE_VERSION:=v$(PKG_VERSION)
+PKG_MIRROR_HASH:=af63df7f88c1031aa5a438d175d77ca9c976e48eb55ff78a6ff6d2bd7272ac51
```

Comment:
Why we are not using tagged releases?

Reply:
Because there is no release since 0.12.5 that is too old to build with kernel5.x